src/helpers/steemitHtmlReady.js

```diff
@@ -241,6 +244,25 @@ function linkify(content, mutate, hashtags, usertags, images, links) {
   return content;
 }
 
+function embedable(child, links, images) {
```

Comment:
Boolean functions better to start with is like `isEmbedable()` also it's better to explain the process in a comment since the function is a little unclear.